jicofo/rootfs/defaults/jicofo.conf

```diff
@@ -3,6 +3,7 @@
 {{ $AUTH_TYPE := .Env.AUTH_TYPE | default "internal" }}
 {{ $ENABLE_RECORDING := .Env.ENABLE_RECORDING | default "0" | toBool }}
 {{ $ENABLE_OCTO := .Env.ENABLE_OCTO | default "0" | toBool }}
+{{ $ENABLE_AUTO_LOGIN := .Env.ENABLE_AUTO_LOGIN | default "1" | toBool }}
```

Comment:
What is the current default? We should preserve that.

Reply:
The current default value is _true_, that's why I define _1_ as default.